timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -185,9 +185,10 @@ def _is_gpu_available(self) -> bool:
     @property
     def ag_default_config(self) -> Dict[str, Any]:
         """The default configuration of the model used by AutoGluon if the model is one of those
-        defined in MODEL_CONFIGS. For now, these are ``amazon/chronos-t5-*`` family of models.
+        defined in MODEL_CONFIGS. For now, these are ``autogluon/chronos-t5-*`` family of models.
         """
-        return MODEL_CONFIGS.get(self.model_path, {})
+        model_name = str(self.model_path).split("/")[-1]
```

Comment:
if the intention is to make it work also on local paths, this will not work for Windows.

Reply:
The intention is to make this work with both `amazon/chronos-t5-*` and `autogluon/chronos-t5-*` paths, so should be fine.

Reply:
Then note that this will now include local paths on *nix that are named according to the amazon-t5-* convention, but not on windows.